elixir/parameter/temp.py

```diff
@@ -147,7 +213,7 @@ def transform_input(self_module, inputs):
         input_list = list()
         for t in inputs:
             if isinstance(t, torch.Tensor):
-                t = to_outplace_tensor(t)
+                t = OutplaceTensor(t)
```

Comment:
I would suggest to rectify the usage of `outplace` here. Please note that `outplace` is not the opposite of `inplace` in English. This can be done in a separate PR.